Anova.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import glob
+import sys
+import json
+import scipy.stats as stats
+import statsmodels.stats.multicomp as multi
+from pathlib import Path
+
+FIGURES_FOLDER = "./Figures"
+ITERATIONS = [100, 500, 2500, 12500]
+
+_, operation, query, datatype = sys.argv
+
+filenames = glob.glob(f"./Results/*/{operation}/*-{operation}-{query}-*.json")
+
+data = {
+  "100": {},
+  "500": {},
+  "2500": {},
+  "12500": {}
+}
+
+if len(filenames) != 12: 
+   raise Exception(f"Data files insufficient") 
+
+def anova(*data):
+  statistic, p_value = stats.f_oneway(data[0], data[1], data[2])
+
+  print(f"ANOVA statistic {str(statistic)} and p-value {p_value}")
+  
+  return p_value < statistic
+
+for file in filenames:
+    with open(file) as infile:
+      lib, op, qu, its = Path(file).stem.split("-")
+
+      lib_data = json.load(infile)
+
+      data[its][lib] = lib_data[datatype]
+
+def tukey(*data):
+  groups = ['MikroORM', 'Prisma', 'Sequelize']
+  df = pd.DataFrame()
+
+  df[groups[0]] = data[0]
```

Comment:
👎🏻 
loop here instead, but it works so its fine

Reply:
ye